docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -332,7 +332,7 @@ The peak number of bytes of memory ever used by the model.
 
 ==== _Data delay has occurred_
 
-`context.message`::
+`context.message` ^*^::
 A preconstructed message for the rule.
 
 `context.results`::
```

Comment:
with `Delayed data` there are 2 possible cases with recovered alerts:
1. No delayed data at all, hence `context.results` is empty.
2. A certain number of docs have been missed, but it hasn't exceeded the `docCount` threshold configured by the user. In that case `context.results` is the same as for the active alert.